lib/optimized/scorecard_fix.dart

```diff
@@ -2,7 +2,8 @@
 // widget directly can avoid rebuilding its sub-tree in every frame of the
 // animation.
 
-import 'package:flutter/animation.dart';
+
```

Comment:
super nit: a bunch of these files have unnecessary new lines added to the top of the files